pkg/limiter/qps_limiter.go

```diff
@@ -65,6 +66,7 @@ func (l *qpsLimiter) UpdateQPSLimit(interval time.Duration, limit int) {
 		l.ticker = time.NewTicker(interval)
 		go l.startTicker()
 	}
+	l.resetTokens()
```

Comment:
resetTokens should before go startTicker()

Reply:
thanks